rfc/0058-error-handling.md

```diff
@@ -317,6 +317,54 @@ A Note on IDs: The IDs in this RFC are only for organisational purposes and MUST
 
 * Raised when decoding of the data into the user object failed
 
+### 21 RateLimitingFailure
+
+* Raised when a service decides that the caller must be rate limited due to exceeding a rate threshold of some sort.
+* Note that since there are many different reasons why a request is rate limited, the error context MUST include the reason / specific type of rate limiting cause for debugging purposes.
+
+Maps to:
+
+* KeyValue
+  * 0x30 RateLimitedNetworkIngress
+  * 0x31 RateLimitedNetworkEgress
+  * 0x32 RateLimitedMaxConnections
+  * 0x33 RateLimitedMaxCommands
+* Cluster Manager (body check tbd)
```

Comment:
Isn't the ClusterManager a SDK2 concept? In SDK3 we broke into different managers for each service:
UserManager Users();
BucketManager Buckets();
QueryIndexManager QueryIndexes();
AnalyticsIndexManager AnalyticsIndexes();
SearchIndexManager SearchIndexes();

https://github.com/couchbaselabs/sdk-rfcs/blob/master/rfc/0054-sdk3-management-apis.md#general-design

Reply:
@jeffrymorris the cluster manager is not a API in this case, but a service (it's on the same level as "keyvalue", "query", "search" ... ) .. in this case it means the service running on 8091/18091